cbatticon.c

```diff
@@ -696,6 +696,16 @@ static void reset_battery_time_estimation (void)
  * tray icon functions
  */
 
+static void set_icon(GtkStatusIcon *tray_icon, const gchar* name) {
+    GdkPixbuf *pix = gtk_icon_theme_load_icon(gtk_icon_theme_get_default(),
+                                              name,
+                                              32,
+                                              GTK_ICON_LOOKUP_USE_BUILTIN,
```

Comment:
@johannesk, what's the reason to use that specific flag?

Reply:
I wrote that code several years ago and do not remember the reason for this flag. Removing that flag yields no observable difference for me. The IconLookupFlags are documented at https://docs.gtk.org/gtk3/flags.IconLookupFlags.html.